### PR TITLE
Add coveralls coverage.

### PR DESCRIPTION
**Pull request by andorp**

### CODE REVIEW

**Comment by andorp** [comment]
Yay! https://coveralls.io/builds/5180717

**Comment by Mikolaj** [comment]
yay, indeed :)  LGTM

File: travis/docker-build.sh [hunk 1]
```diff
@@ -12,12 +12,12 @@ cabal sandbox init --sandbox=$CABAL_SANDBOX
...
 # Install from the current code
 cabal install --enable-tests --only-dependencies --reorder-goals
-cabal configure --enable-tests --disable-optimization
+cabal configure --enable-tests --disable-optimization --enable-benchmarks --enable-coverage
 cabal build --ghc-options="$GHC_OPTIONS"
 
 # Test
 set +e
-cabal test --show-details=never --ghc-options="$GHC_OPTIONS"
+run-cabal-test spec --show-details=never
```
Comment (fisx) [inline travis/docker-build.sh]: wondering where this is coming from.  @andorp doesn't know either.  not essential though, it works!

**Comment by fisx** [comment]
thanks for this!  now we just need to improve our stats...

**Comment by andorp** [comment]
We will do that in time.

**Pull request merged**